be/test/exec/unix_odbc_test.cpp

```diff
@@ -0,0 +1,28 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "odbc/sql.h"
```

Comment:
```suggestion
#include "sql.h"
```

Reply:
may be <odbc/sql.h> or <sql.h>

Reply:
`sql.h` is in `thirdparty/installed/included`, so it should be `"sql.h"`

Reply:
I think only Doris own code use `""`, for third-party code use `<>`.
Glog, protobuf are all in the thirdparty, however we inlcude them with `""`

Reply:
Oh, I misunderstood.
yes, it should be `<>`.